### PR TITLE
Quantize QuestionAnswering models

### PR DESCRIPTION
**Pull request by bgawrych**
## Description ##
This PR enables quantization on question answering scripts.
Added custom calibration collector to avoid significant accuracy drop

### CODE REVIEW

**Review by bartekkuncer — COMMENTED** [review]
Looks good to me.

File: scripts/question_answering/models.py [hunk 1]
```diff
@@ -287,10 +289,14 @@ def forward(self, tokens, token_types, valid_length, p_mask, start_position):
             Shape (batch_size, sequence_length)
         answerable_logits
         """
+        backbone_net = self.backbone
+        if self.quantized:
```
Comment (anko-intel) [inline scripts/question_answering/models.py:293]:
```suggestion
        if self.quantized_bacbone is not None:
```
end remove quantized ?

**Review by anko-intel — APPROVED** [review]

File: scripts/question_answering/models.py (hunk 1 above)
Comment (bgawrych) [reply scripts/question_answering/models.py:293]: I thought about it, but I remained this quantized flag as switch on/off of quantized model - not sure if it is really usable. What do you think?

**Review by bartekkuncer — APPROVED** [review]
